components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/constants/OpenAPIConstants.java

```diff
@@ -15,4 +20,25 @@
     public static final String THROTTLING_TIER = "x-wso2-throttling-tier";
     public static final String DISABLE_SECURITY = "x-wso2-disable-security";
     public static final String AUTHORIZATION_HEADER = "x-wso2-auth-header";
+
+    /**
+     * API security types supported by mgw
+     */
+    public enum APISecurity {
+        basic,
+        oauth2,
+        jwt,
+        apikey
+    }
+    public static final String SWAGGER_X_WSO2_API_SECURITY = "x-wso2-api-security";
```

Comment:
Move this to line 23 and rename to `API_SECURITY` to match with other similar constants.

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/965/commits/bf2732d5724d4fecee0ab331a1f7ef6034b327b5. Thanks.